theano/sparse/basic.py

```diff
@@ -1346,6 +1346,23 @@ def perform(self, node, inputs, outputs):
         assert _is_sparse(x)
         out[0] = x[start1:stop1:step1, start2:stop2:step2]
 
+    def grad(self, inputs, grads):
+        x = inputs[0]
+        rest = inputs[1:]
+        gz, = grads
+
+        assert _is_sparse_variable(x) == True
+
+        return [x.zeros_like() + gz] + [DisconnectedType()()] * len(rest)
```

Comment:
I mean, make a new op like IncSubtensor, but for this Op and use it in this op grad like Subtensor.grad.

Mostly, do like Subtensor/IncSubtensor, but on sparse data. But don't forget that Subtensor is much more complicated then this op, so don't try to replicate everything from Subtensor.